geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -35,6 +35,7 @@ object HBaseQueryPlan {
     explainer.pushLevel(s"${prefix}Plan: ${plan.getClass.getName}")
     explainer(s"Table: ${Option(plan.table).orNull}")
     explainer(s"Ranges (${plan.ranges.size}): ${plan.ranges.take(5).map(rangeToString).mkString(", ")}")
+    explainer(s"Remote Filters: ${plan.remoteFilters.map(f => f.getClass.getCanonicalName).mkString(", ")}")
```

Comment:
true, good point. maybe we should do name + `toByteArray`? that seems to be the equivalent of the accumulo `IteratorSetting`, although not sure how readable it will be.